openfe/tests/storage/conftest.py

```diff
@@ -1 +1,4 @@
-from gufe.tests.conftest import *
+from gufe.tests.conftest import (
+    absolute_transformation, complex_equilibrium,
```

Comment:
Fun story, benzene_modifications is session scoped, and rdkit mols in gufe and smallmoleculecomponents in openfe.

Because of the way the discovery order works locally vs when doing it off an installed package, we weren't picking up issues related to that one fixture being overriden until now.

Reply:
amazing catch

Reply:
would be if I could actually fix it :(